docs/infrastructure/README.md

```diff
@@ -0,0 +1,167 @@
+# Dotkom Container Service (Evergreen)
+
+Evergreen is our container service that should be used to run all Monoweb applications in all environments, including
+production and staging. Evergreen is based on Docker containers, and we use Docker Compose as the orchestrator.
+
+At its core, Evergreen is a single Traefik load balancer, and a set of Docker containers running the various apps that
+are part of Monoweb. To keep simplicity low, Evergreen is hosted on a single AWS EC2 instance. Because of this, the
+services are not intended to be highly scalable, and should likely be used for smaller applications that we don't mind
+small downtimes for.
+
+**Table of Contents**
+
+- [Motivation](#motivation)
+- [Service Architecture](#service-architecture)
+- [Deployment & Scaling](#deployment--scaling)
+- [Pricing](#price)
+- [Pitfalls](#pitfalls--alternatives)
+
+## Motivation
+
+The motivation for Evergreen is to provide a really cheap way to run multiple small applications that can tolerate
+occasional downtime. Additionally, we want to use Docker as the deployment format. Furthermore, Docker Compose is
+relatively simple, and we want to keep deployment complexity low in order to make it easy for new developers to
+understand.
+
+The proposed solution has a set of pitfalls outlined in the [Pitfalls & Alternatives](#pitfalls--alternatives) section,
+but we believe that keeping things simple and cheap is rather important for a student organization with limited
+financial resources.
+
+## Service Architecture
+
+The entire application fleet is hosted on a single AWS EC2 (t4g.medium as of writing) instance. The instance is
+available to the public internet through a VPC, but all traffic should be routed from CloudFront to Traefik. Traefik is
```

Comment:
> all traffic should be routed from CloudFront to Traefik.

Given the intended audience (which includes 1st graders) this should have more ELI5 of _why_ it is smart to route it through CloudFront first

Reply:
Good point